tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/ManualInstrumentation/Configuration/TracerSettings/StatsComputationEnabledGetIntegration.cs

```diff
@@ -0,0 +1,35 @@
+﻿// <copyright file="StatsComputationEnabledGetIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Telemetry;
+using Datadog.Trace.Telemetry.Metrics;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.ManualInstrumentation.Configuration.TracerSettings;
+
+/// <summary>
+/// System.Boolean Datadog.Trace.Configuration.TracerSettings::get_StatsComputationEnabled() calltarget instrumentation
+/// </summary>
+[InstrumentMethod(
+    AssemblyName = "Datadog.Trace.Manual",
+    TypeName = "Datadog.Trace.Configuration.TracerSettings",
+    MethodName = "get_StatsComputationEnabled",
+    ReturnTypeName = ClrNames.Bool,
+    ParameterTypeNames = [],
+    MinimumVersion = ManualInstrumentationConstants.MinVersion,
+    MaximumVersion = ManualInstrumentationConstants.MaxVersion,
+    IntegrationName = ManualInstrumentationConstants.IntegrationName)]
+[Browsable(false)]
+[EditorBrowsable(EditorBrowsableState.Never)]
+public class StatsComputationEnabledGetIntegration
+{
+    internal static CallTargetState OnMethodBegin<TTarget>(TTarget instance)
+    {
+        TelemetryFactory.Metrics.Record(PublicApiUsage.TracerSettings_StartupDiagnosticLogEnabled_Get);
```

Comment:
Looks like this metric should be pointing to `StatsComputationEnabled_Get` (if there is one)

Reply:
oops, good catch 👍